src/frontends/pytorch/src/op/convolution.cpp

```diff
@@ -11,42 +11,85 @@ namespace frontend {
-                                                     pads_begin,
-                                                     pads_end,
-                                                     dilations);
+        if (!transposed) {
+            conv = context.mark_node(std::make_shared<opset8::Convolution>(context.get_input(0),
+                                                                           context.get_input(1),
+                                                                           strides,
+                                                                           pads,
+                                                                           pads,
+                                                                           dilations));
+        } else {
+            conv = context.mark_node(std::make_shared<opset8::ConvolutionBackpropData>(context.get_input(0),
+                                                                                       context.get_input(1),
+                                                                                       strides,
+                                                                                       pads,
+                                                                                       pads,
+                                                                                       dilations,
+                                                                                       ov::op::PadType::EXPLICIT,
+                                                                                       output_padding));
+        }
     } else {
-        conv = std::make_shared<opset8::GroupConvolution>(
-            context.get_input(0),
-            context.mark_output(reshape_kernel_for_group(context, context.get_input(0), context.get_input(1), groups)),
-            strides,
-            pads_begin,
-            pads_end,
-            dilations);
+        if (!transposed) {
+            conv = context.mark_node(std::make_shared<opset8::GroupConvolution>(
+                context.get_input(0),
+                context.mark_output(
+                    reshape_kernel_for_group(context, context.get_input(0), context.get_input(1), groups)),
+                strides,
+                pads,
+                pads,
+                dilations));
+        } else {
+            conv = context.mark_node(std::make_shared<opset8::GroupConvolutionBackpropData>(
+                context.get_input(0),
+                context.mark_output(
+                    reshape_kernel_for_group(context, context.get_input(0), context.get_input(1), groups)),
+                strides,
+                pads,
+                pads,
+                dilations,
+                ov::op::PadType::EXPLICIT,
+                output_padding));
+        }
+    }
+    if (!context.input_is_none(2)) {
+        auto bias = context.get_input(2);
+        auto bias_rank = bias.get_partial_shape().rank();
```

Comment:
If `bias` has dynamic shape or rank this will fail, but can we support it?

Reply:
I thought about it, but for me such situation looks dubiously...